src/less/filter-button/filter-button.less

```diff
@@ -38,19 +38,14 @@ a.filter-link:visited {
 
 button.filter-button[aria-pressed="true"],
 a.filter-link--selected {
-    .background-color-token(filter-button-selected-background-color, color-background-inverse);
-    .color-token(filter-button-selected-foreground-color, color-foreground-on-inverse);
+    border: 1px solid;
     font-weight: bold;
-
-    &:focus,
-    &:hover,
-    &:active {
-        background-color: var(--color-state-inverse-hover);
-    }
+    padding: calc(@spacing-100 - 1px) calc(@spacing-200 - 1px);
```

Comment:
No need to do these calculations to exclude the border. Just use `box-sizing: border-box;` to include the padding and border in the width.

Reply:
Also, don't forget to do the border color difference for dark mode.